Jobs.Fetcher.Facebook/Client/ApiManager.cs

```diff
@@ -106,7 +107,7 @@ public class ApiManager {
             HttpResponseMessage response;
 
             //Setting up a random difference for each sleep for jobs not to run concurrently
-            var milliSecondsDelay = RequestDelay * ((new Random()).Next(0, 500) + 750);
+            var milliSecondsDelay = RequestDelay * ((new Random()).Next(0, 500) + 750) / 1000;
```

Comment:
I don't understand why it was divided by 1000. That would make it seconds, not ms, right? And Thread.Sleep takes milliseconds